src/bin/mosaic-start.ts

```diff
@@ -26,24 +27,28 @@ mosaic
       password,
     } = NodeOptions.parseOptions(options);
 
-    for (const chainId of chainIds) {
-      const node: Node = NodeFactory.create({
-        chainId,
-        mosaicDir,
-        port,
-        rpcPort,
-        websocketPort,
-        keepAfterStop,
-        unlock,
-        password,
-      });
+    const chainId = NodeFactory.getChainId(chain);
+    if (port === undefined) {
+      port = NodeOptions.getPortForChainId(chainId);
+    }
+    if (rpcPort === undefined) {
+      rpcPort = NodeOptions.getRpcPortForChainId(chainId);
+    }
+    if (websocketPort === undefined) {
+      websocketPort = NodeOptions.getWsPortForChainId(chainId);
+    }
 
-      node.start();
+    const node: Node = NodeFactory.create({
+      chainId: chain,
```

Comment:
> This is confusing. Now we have `chain` and `chainId` meaning different things. Can that be clarified somehow? How much effort would it be to rename what is currently called `chainId` to `chainName`?

Yes, it is confusing, I have reverted the name to `chainId`. I tried to change rename the `chainId` to `chain`/`chainName` but it will take some effort to carefully change all the instances and test it carefully. Maybe we should create a new followup ticket. In that way, the changes will be clean, and this ticket will focus on port changes. 

> 
> Also, does this change mean that `testnet` and `dev` are now run with parity and without the settings from the local config??

Yes, removed `testnet` and `dev` form `officialIdentifiers` list.